### DISCUSSION TITLE
xeyes 8-) Mac ELF for testing darling

### DISCUSSION THREAD

**Discussion by john-peterson** [darlinghq/darling#1394] [General]
Notepad.exe xclock glxgears. Most used programs I have ever used. I'm obsessed with translation layers and I'm I have darling in jammy. But I have no programs to run

Anyone can post links without terrifying the mods. Just the most generic uncontroversial Mac program

{{{((([[[This doesn't matter for anyone but I will write it anyway. My goal today is to run `prl_disk_tool` if it's in the open domain. I need to convert parallels images to Linux. I feel bad about it no one will make an money. So this is all in paranthesis. Only interesting to me)))]]]}}}}

*Edit I forgot about xeyes. Preferably a single file app. Or does all apps come as folders?

**Comment by john-peterson** [comment]
https://github.com/XQuartz/XQuartz/releases/download/XQuartz-2.8.5/XQuartz-2.8.5.pkg

But I had trouble finding any program

```
sudo apt install libfuse-dev libbz2-dev libicu-dev libxml2-dev libssl-dev libz-dev pkg-config cmake autoconf

https://github.com/tpoechtrager/xar

xar -xf XQuartz-2.8.5.pkg -C xquartz

dev@emu:~$ tree xquartz/
xquartz/
├── Distribution
├── Resources
│   ├── LICENSE.txt
│   └── README.txt
└── XQuartzComponent.pkg
    ├── Bom
    ├── PackageInfo
    ├── Payload
    └── Scripts

3 directories, 7 files
dev@emu:~$

dev@emu:~$ gunzip xquartz/XQuartzComponent.pkg/Payload
gzip: xquartz/XQuartzComponent.pkg/Payload: unknown suffix -- ignored

**Comment by bugaevc** [comment]
To install a PKG, don't just unxar it, use `installer` -- please [Installing software](https://docs.darlinghq.org/installing-software.html#pkg-files) in the docs.

It's unlikely that XQuartz is going to run. You may have more luck with running xeyes against the host's X11 server (which is the one `DISPLAY` already points to anyway).
